contracts/src/v0.8/dev/automation/2_1/KeeperRegistryBase2_1.sol

```diff
@@ -99,8 +74,10 @@ abstract contract KeeperRegistryBase2_1 is ConfirmedOwner, ExecutionPrevention {
   mapping(uint256 => Upkeep) internal s_upkeep; // accessed during transmit
   mapping(uint256 => address) internal s_upkeepAdmin;
   mapping(uint256 => address) internal s_proposedAdmin;
-  mapping(uint256 => bytes) internal s_checkData;
+  mapping(uint256 => bytes) internal s_pipelineData;
```

Comment:
nit. having 'check' in the name makes it clear where it is used

Reply:
not exclusively used in check anymore